.github/workflows/bazel.yml

```diff
@@ -614,7 +617,9 @@ jobs:
         uses: jfrog/setup-jfrog-cli@d0a59b1cdaeeb16e65b5039fc92b8507337f1559 # pin@v3
 
       - name: Set dry run environment variable
-        if: ${{ github.event_name != 'push' || github.repository_owner != 'magma' || github.ref_name != 'master' }}
+        if: |
+          ${{ github.event_name != 'push' || github.repository_owner != 'magma' ||
+          ( github.ref_name != 'master' && ! startsWith(github.ref_name, 'v1.') ) }}
```

Comment:
Lgtm, but from the test runs we can't tell if all the cases are working as intended. I assume you changed more than the `repository_owner` check for the test runs? Because dry run is set for both runs in your tests, even though they are supposed to be `master` and release branches?

Reply:
I tested the workflow again. For the tests I removed all steps requiring (magma) credentials and changed `github.repository_owner != 'mpfirrmann'`. One test was done with `github.ref_name != 'master'`, the other one to test the if statement with `github.ref_name == 'master'` in the condition for the dry run.
The [first test](https://github.com/mpfirrmann/magma/actions/runs/4294272225/jobs/7483102661) skipped the step, while the [second test](https://github.com/mpfirrmann/magma/actions/runs/4296786173/jobs/7488945204) did execute the dry run step.

Reply:
Thanks for running the tests again!